streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/StatefulProcessorNode.java

```diff
@@ -17,28 +17,30 @@
 
 package org.apache.kafka.streams.kstream.internals.graph;
 
+import org.apache.kafka.streams.processor.StateStore;
 import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder;
-import org.apache.kafka.streams.state.KeyValueStore;
 import org.apache.kafka.streams.state.StoreBuilder;
 
 import java.util.Arrays;
 
 public class StatefulProcessorNode<K, V> extends StatelessProcessorNode<K, V> {
 
     private final String[] storeNames;
-    private final StoreBuilder<KeyValueStore<K, V>> storeBuilder;
+    private final StoreBuilder<? extends StateStore> storeBuilder;
```

Comment:
By properly forwarding the type parameters through to this point, I was able to determine that we were previously assuming the wrong type for this field!

Luckily, we haven't really been using it yet.

Reply:
In this case do we still need the parameterized `K, V` then?